### DISCUSSION TITLE
yo

### DISCUSSION THREAD

**Discussion by ttodosi** [DeabLabs/cannoli#33] [General]
Yo - just throwing my hat in the ring. I saw your comments on the Professor Synapse videos. This app is _killer_ for me and I haven't found anything remotely as powerful and easy to use in the AI space. Looking forward to contributing some of my node setups and ideas. Can't wait to see how ollama support with API action and 'for each' come together. Thanks!

**Comment by blindmansion** [comment]
Thank you, means a lot. Would love to see any cannolis you want to share!

Ollama is implemented as of last week, you can find the instructions for it in the readme, let me know if you have any questions about how to set it up!

For each groups is a lot closer than they were a month ago, still percolating on them for sure.

**Comment by ttodosi** [comment]
...what is this smart-connections integration brewing!!!!? 🥳time for affogato!!!

**Reply by blindmansion** [reply]
Its up now!

**Reply by ttodosi** [reply]
!!!! :_carefully connects one crema cannolo to 5 pistachio cannoli_

Ho avuto un'illuminazione!!!!

**Reply by arminta7** [reply]
Which version of smart connections is this working with?

**Reply by blindmansion** [reply]
I tested it on 2.1.65 and 66 and it was working. What version are you on?

**Reply by arminta7** [reply]
Thanks. I've been bouncing around SC versions because I was having some trouble (unrelated to Cannoli). I was able to get everything to work on the version you mentioned. Very cool! Lots I want to do with this.

**Reply by blindmansion** [reply]
Ok cool! It wouldn't be bad to make cases for different versions, so let me know if you're still running into issues with it.

**Comment by ttodosi** [comment]
I’m on 2.2.14 The last cannoli college with 4 queries  was giving me
errors, but I think it was something to do with the content of my notes. It
works This morning.

**Reply by blindmansion** [reply]
Oh is that a beta version of smart connections? Let me know if you run into more issues with it, I may have to add some cases for different versions
